src/crud/useWavesurferRegions.tsx

```diff
@@ -82,6 +94,7 @@ export function useWaveSurferRegions(
             400
           ).then(() => {
```

Comment:
If the wait for it fails, should the current region be set to something to show there is no current region set?